### PR TITLE
NEOR-17 csharp-script : add reference assembly to oracle connector

### PR DESCRIPTION
**Pull request by yyang-convey**

### CODE REVIEW

File: csharp-code-evaluator-ut/CSharpCodeEvaluatorTestClass/CSharpCodeEvaluatorTestClass.csproj [hunk 1]
```diff
@@ -37,6 +37,10 @@
     <AssemblyOriginatorKeyFile>CSharpCodeEvaluatorTestClass.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="oracle.dataaccess, Version=4.112.4.0, Culture=neutral, PublicKeyToken=89b483f429c47342, processorArchitecture=AMD64">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\Windows\Microsoft.NET\assembly\GAC_64\Oracle.DataAccess\v4.0_4.112.4.0__89b483f429c47342\oracle.dataaccess.dll</HintPath>
```
Comment (jsbattig) [inline csharp-code-evaluator-ut/CSharpCodeEvaluatorTestClass/CSharpCodeEvaluatorTestClass.csproj]: @yyang-convey , you should change this putting the odp.net library on the dependencies folder, as we did with NUnit. You could in fact have a subfolder for x64 and another one for x86

File: csharp-code-evaluator-ut/csharp-script-ut.csproj [hunk 2]
```diff
@@ -38,6 +38,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\dependencies\nunit.framework.dll</HintPath>
     </Reference>
+    <Reference Include="oracle.dataaccess, Version=4.112.4.0, Culture=neutral, PublicKeyToken=89b483f429c47342, processorArchitecture=AMD64">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\Windows\Microsoft.NET\assembly\GAC_64\Oracle.DataAccess\v4.0_4.112.4.0__89b483f429c47342\oracle.dataaccess.dll</HintPath>
```
Comment (jsbattig) [inline csharp-code-evaluator-ut/csharp-script-ut.csproj]: @yyang-convey , you should change this putting the odp.net library on the dependencies folder, as we did with NUnit. You could in fact have a subfolder for x64 and another one for x86

File: csharp-code-evaluator-ut/CSharpCodeEvaluatorTestClass/CSharpCodeEvaluatorTestClass.csproj (hunk 1 above)
Comment (yyang-convey) [inline csharp-code-evaluator-ut/CSharpCodeEvaluatorTestClass/CSharpCodeEvaluatorTestClass.csproj]: @jsbattig the configuration is "Any CPU" currently, should we introduce x86 and x64 configuration builds?

**Comment by jsbattig** [comment]
Closing. All of this really belongs to the caller code not to the library

**Pull request closed without merging**